packages/components/psammead-numbered-list/src/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,101 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PsammeadNumberedList should render correctly from ltr 1`] = `
+.c0 {
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  margin-top: 0;
+}
+
+.c1 {
+  margin-bottom: 1rem;
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c0 {
+    font-size: 1rem;
+    line-height: 1.375rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c0 {
+    font-size: 1rem;
+    line-height: 1.375rem;
+  }
+}
+
+<ol
+  class="c0"
+  dir="ltr"
+>
+  <li
+    class="c1"
+  >
+    First item on the list
+  </li>
+  <li
+    class="c1"
+  >
+    Second item on the list
+  </li>
+  <li
+    class="c1"
+  >
+    Final list item
+  </li>
+</ol>
+`;
+
+exports[`PsammeadNumberedList should render correctly from rtl 1`] = `
+.c0 {
+  font-size: 1.125rem;
+  line-height: 1.75rem;
+  font-family: "BBC Reith Qalam",Arial,Verdana,Geneva,Helvetica,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  margin-top: 0;
+}
+
+.c1 {
+  margin-bottom: 1rem;
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c0 {
+    font-size: 1.375rem;
+    line-height: 2rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c0 {
+    font-size: 1.375rem;
+    line-height: 2rem;
+  }
+}
+
+<ol
```

Comment:
Not sure if you are adding your own list numbers as per https://bbc.github.io/psammead/?path=/story/components-mostread-list--default or if you are using the default CSS list ordering numbers, if adding your own numbers you may need an attribute of role="list" on the ul so VoiceOver Mac and iOS users get list semantics.

Reply:
This was one of the issues I wanted to discuss. I saw the Most Read implementation but I noticed the translations only go up to 10, so not sure how we'd handle longer lists.  Can we not rely on browsers to provide correct numerals or do they not translate them? Happy to discuss on Slack if that's easier.

Reply:
Think we add our numbers as we wanted different styling, that's not possible if you use standard CSS list numbers...

Reply:
If you don't need different styling I would just use the standard CSS list numbers provided they work ok with AT...